opentelemetry-sdk/src/opentelemetry/sdk/_metrics/aggregation.py

```diff
@@ -156,15 +156,17 @@ def __init__(
         record_min_max: bool = True,
     ):
         super().__init__()
-        # pylint: disable=unnecessary-comprehension
-        self._boundaries = [boundary for boundary in (*boundaries, inf)]
-        self.value = [0 for _ in range(len(self._boundaries))]
+        self._boundaries = tuple(boundaries)
+        self._bucket_counts = self._new_explicit_bounds()
         self._min = inf
         self._max = -inf
         self._sum = 0
         self._record_min_max = record_min_max
         self._start_time_unix_nano = _time_ns()
 
+    def _new_explicit_bounds(self) -> List[int]:
```

Comment:
```suggestion
    def _get_bucket_counts(self) -> List[int]:
```

Reply:
What's the reasoning here? it doesn't get the bucket counts, it returns a new zero'ed out list

Reply:
The reasoning is just that the result of this function is being assigned to `self._bucket_counts`. What is an explicit bound?

I edited my suggestion to include empty, maybe that is more clear.

Reply:
Got it, updated